src/document-submit.js

```diff
@@ -64,7 +67,7 @@ export function onDocumentFormSubmit_(e) {
     user().assert(action,
         'form action attribute is required for method=GET: %s', form);
     assertHttpsUrl(action, dev().assertElement(form), 'action');
-    user().assert(!startsWith(action, urls.cdn),
+    user().assert(!isProxyOrigin(action),
```

Comment:
The problem with this is it might cause a lot of painful errors when someone is building AMP docs locally with a matching URL pattern. Like if my local server is running on `8000` and have a `/c/` or `/v/` scope for my pages, then all of them would appear to error although they shouldn't.